plugins/@grouparoo/braze/src/lib/export/destinationMappingOptions.ts

```diff
@@ -22,9 +22,9 @@ export const destinationMappingOptions: DestinationMappingOptionsMethod =
           { key: "phone", type: "phoneNumber", important: false },
           { key: "gender", type: "string", important: false },
           { key: "country", type: "string", important: false },
-          { key: "date_of_first_session", type: "string", important: true },
-          { key: "date_of_last_session", type: "string", important: true },
-          { key: "dob", type: "string", important: true },
+          { key: "date_of_first_session", type: "date", important: true },
```

Comment:
These should actually stays a strings. Current `"date"` actually means `new Date()` which is really a `"datetime"`